Corona-Warn-App/src/main/res/drawable/splash_screen.xml

```diff
@@ -4,9 +4,4 @@
     <!-- background with solid color -->
     <item android:drawable="@color/colorBackground" />
 
-    <!-- logo -->
-    <item
-        android:drawable="@drawable/ic_splash_logo_round"
-        android:gravity="center" />
-
```

Comment:
does this mean there is no logo on below Android M devices splash?

Reply:
Yes, otherwise it will look similar to this (see https://github.com/corona-warn-app/cwa-app-android/pull/2955/commits/d7c66b66f35704c592b0cceff440933664ebacd1):
![Screenshot_1619782755](https://user-images.githubusercontent.com/16943720/116690362-917f1200-a9b9-11eb-9193-8edb6aeb7131.png)

Reply:
I think this will work, can you please try on your side?
```xml
<?xml version="1.0" encoding="utf-8"?>
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">

    <!-- background with solid color -->
    <item android:drawable="@color/colorBackground" />

    <item>
        <bitmap
            android:gravity="center"
            android:src="@drawable/ic_splash_logo_round" />
    </item>

</layer-list>
```

I tired it looks like this:
<img width="527" alt="Screenshot 2021-04-30 at 15 59 55" src="https://user-images.githubusercontent.com/25054729/116705955-656d8c00-a9cd-11eb-9c1c-a123a0b5357a.png">

Reply:
Thanks, it looks remarkably similar to what was in the file originally but your solution fixes the problem :tada: will integrate into PR & remember for the future.

Reply:
I found no disadvantage in doing it this way, therefore I removed the separation of assets between Android versions again.